pkg/resourceapply/apps_test.go

```diff
@@ -47,7 +47,11 @@ func TestApplyStatefulSet(t *testing.T) {
 			Spec: appsv1.StatefulSetSpec{
```

Comment:
empty selector is invalid sts

https://github.com/kubernetes/kubernetes/blob/c3384191ea6410121339a48d1c68ba6f4b3ab31a/pkg/apis/apps/validation/validation.go#L139-L143

Reply:
done